docs/eips/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ethereum Improvement Proposals (EIPs)
+meta:
+  - name: description
+    content: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+  - property: og:title
+    content: Ethereum Improvement Proposals (EIPs) | Ethereum.org
+  - property: og:description
+    content: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+lang: en-US
+sidebar: auto
+---
+
+# Introduction to Ethereum Improvement Proposals (EIPs)
+
+## What is an EIP?
```

Comment:
Small nit: perhaps we should use the plural there (What are EIPs?) given we use it in the sentence below, or vice-versa.